QT/mainwindow.cpp

```diff
@@ -31,8 +30,26 @@ void MainWindow::on_FrequencySlider_valueChanged(int value) {
     ui->FrequencyLabel->setText(QString::number(value) + "Hz");
 }
 
+void MainWindow::drawWaveFromFile(QString file)
+{
+    ui->timeDomainInput->setSource(file);
+    ui->timeDomainInput->plot();
+}
+
+void MainWindow::on_playButton_clicked(bool)
+{
+
+    // Get current cursor position when it exists
+    //int position = 0;
+    //
+    std::cout << "Play Button Pushed" << std::endl;
+    QString file = "C:/Users/The_M/Documents/repos/ECE496/ImperialMarch60.wav";
```

Comment:
Pls look into using QDir::currentPath() with the fix I showed

Reply:
added